### PR TITLE
Fix unit test failures

### PR DESCRIPTION
**Pull request by cssuh**
Fixes the failing unit tests in our build

### CODE REVIEW

File: test/connectionProfile.test.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@
  * ------------------------------------------------------------------------------------------ */
 
 'use strict';
+import vscode = require('vscode');
```
Comment (kburtram) [inline test/connectionProfile.test.ts:7]: why not use `import * as vscode from 'vscode';` syntax for this import?

**Review by abist — APPROVED** [review]

File: test/connectionProfile.test.ts [hunk 2]
```diff
@@ -14,9 +15,10 @@ import { ConnectionUI } from '../src/views/connectionUI';
 import { ConnectionStore } from '../src/models/connectionStore';
 import ConnectionManager from '../src/controllers/connectionManager';
 import VscodeWrapper from '../src/controllers/vscodeWrapper';
-
+import Constants = require('../src/constants/constants');
```
Comment (kburtram) [inline test/connectionProfile.test.ts:18]: same here.  I think in typescript we generally prefer an `import..from` variation to calling require directly.  Is there a reason to use require directly here?

**Review by kburtram — APPROVED** [review]
LGTM

File: test/connectionProfile.test.ts (hunk 2 above)
Comment (cssuh) [reply test/connectionProfile.test.ts:18]: I think I had just copied those imports from elsewhere, I can change them to the `import from` form

**Pull request merged**